irohad/main/irohad.cpp

```diff
@@ -74,9 +88,18 @@ DEFINE_string(keypair_name, "", "Specify name of .pub and .priv files");
 DEFINE_validator(keypair_name, &validate_keypair_name);
 
 /**
- * Creating boolean flag for overwriting already existing block storage
+ * Creating enum flag for overwriting already existing block storage
+ */
+DEFINE_string(
+    blockstore_strategy,
+    "abort",
+    "Strategy for applying genesis block when block store is not empty. Can be "
+    "'overwrite' to rewrite blockstore with genesis, 'continue' to ignore "
+    "genesis block  or 'abort' to abort");
```

Comment:
Not that obvious doc

"continue to ignore genesis block" - will it create a separate blockstore, or just run over the existing one? That's hard to understand from that description

"abort to abort" sounds useless. Abort what? Launching iroha, reading from blockstore or smth else?